web/src/components/molecules/Content/Table/DropdownRender.tsx

```diff
@@ -328,20 +329,31 @@ const DropdownRender: React.FC<Props> = ({
 
       onTableControl(undefined, currentFilters.filter(Boolean));
     } else {
-      searchParams.set("direction", filterOption.current.value === "ASC" ? "ASC" : "DESC");
+      const direction =
+        filterOption.current.value === "ASC" ? SortDirection["Asc"] : SortDirection["Desc"];
+      let fieldType: FieldType;
+      let fieldId = "";
       switch (filter.id as string) {
         case "CREATION_DATE":
         case "CREATION_USER":
         case "MODIFICATION_DATE":
         case "MODIFICATION_USER":
         case "STATUS":
-          searchParams.set("sortFieldType", filter.id);
+          fieldType = filter.id as FieldType;
```

Comment:
This code is confusing? 
Please stop using api types directly, instead create local types and make converter functions to convert from api type to local type.
And try to avoid type assertions as much as possible.
We are using typescript not javascript, so types should be taken in consideration.

Reply:
filter.id is the key of a column that the user wants to filter.
If a column is a meta field or field, then the key of the column is corresponding field ID.
If a column represents the data of item's information (ex: createdAt, createdBy), the key of column is the fieldType of the corresponding data(ex: "CREATION_DATE", "CREATION_USER")